safe-global/safe-wallet-web
# fix: Use chain logos from config service

## What it solves

Resolves #3379

## How this PR fixes it

- Uses `chainConfig.chainLogoUri` as the primary image source

Note: We can delete `getChainLogo` once logos are uploaded to the config service on staging and prod

## How to test it

1. Open the app
2. Open the network selector
3. All logos should still be visible and have the same quality

## Screenshots

<img width="215" alt="Screenshot 2024-03-07 at 12 50 44" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/3a542f26-4f4d-46ab-8764-099134e27417">

## Checklist
